packages/fx-core/src/plugins/resource/apiconnector/constants.ts

```diff
@@ -33,6 +33,8 @@ export class Constants {
   public static readonly pkgLockFile = "package-lock.json";
   public static readonly sdkName = "@microsoft/teamsfx";
   public static readonly sdkJsName = "@microsft/teams-js";
+  public static readonly READ_MORE = "Read more";
```

Comment:
we may also need to localize this string

Reply:
sure, will fix in the later PR.